src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/ElectricGoldPan.java

```diff
@@ -40,31 +40,38 @@ public class ElectricGoldPan extends AContainer implements RecipeDisplayItem {
     private final GoldPan goldPan = SlimefunItems.GOLD_PAN.getItem(GoldPan.class);
     private final GoldPan netherGoldPan = SlimefunItems.NETHER_GOLD_PAN.getItem(GoldPan.class);
 
-    private final ItemStack gravel = new ItemStack(Material.GRAVEL);
-    private final ItemStack soulSand = new ItemStack(Material.SOUL_SAND);
-
     @ParametersAreNonnullByDefault
     public ElectricGoldPan(ItemGroup itemGroup, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
         super(itemGroup, item, recipeType, recipe);
         addItemSetting(overrideOutputLimit);
     }
 
     /**
-     * This returns whether the {@link ElectricGoldPan} will stop proccessing inputs
+     * @deprecated
+     *
+     * @see ElectricGoldPan#isOutputLimitOverridden()
+     */
+    @Deprecated
```

Comment:
We should really add deprecated since but thats also a pain to do.
Since we only care about RC and there is an RC releasing this week and i doubt this will be in it can we add @Deprecated since: "RC 36"

Reply:
example: https://github.com/Slimefun/Slimefun4/blob/75a781d3d651b6db7fe6697ecc48bd28e486cd49/src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/LongFallBoots.java#L28

Reply:
> example: 
> https://github.com/Slimefun/Slimefun4/blob/75a781d3d651b6db7fe6697ecc48bd28e486cd49/src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/LongFallBoots.java#L28

Should I add exactly that?